pkg/resources/manifests_test.go

```diff
@@ -16,51 +16,48 @@ import (
 )
 
 func TestGenerateStatefulSetManifest(t *testing.T) {
-	options := []bool{true, false}
 
-	for _, useHeadlessSvc := range options {
-		t.Run(fmt.Sprintf("generate statefulset with headlessSvc %v", useHeadlessSvc), func(t *testing.T) {
+	t.Run("generate statefulset with headlessSvc", func(t *testing.T) {
```

Comment:
I see GenerateStatefulSetManifest will always include reference to headless svc in servicename